### ISSUE TITLE
Did you think about TDD?

### ISSUE THREAD

**Issue by colachg** [EQuimper/youtube-golang-graphql-tutorial#2]
I mean that if you can add TDD in  your tutorial that will be better :)

**Comment by EQuimper** [comment]
@colachg Hello :) Yes I love TDD, this is something I do when I build client project in Elixir/Laravel/Node. I've done a bit in Go but again like I said in video, I'm not an expert in Go, and this is something I want to do more. I've done table driven table + tdd for golang function. But when it's time to tdd the full integration stuff, I feel it's hard to get my head around all those thing. 

Do that make sense ? But I'm continuing learning go and when I will feel more comfortable will do this in tdd

**Comment by colachg** [comment]
> But when it's time to tdd the full integration stuff, I feel it's hard to get my head around all those thing

Yes test is a design of implement so it's difficult. I have watched your tutorial and am learning golang in practice. Now TDD is what I am thinking about and I don't know how to test so I asked you.

Thanks for your reply and this great tutorial :)

**Comment by EQuimper** [comment]
@colachg perfect I keep that in mind and will try to improve my test skill in go and will share a video with that :) Thank you :) I really appreciate
